doc/conda/meta.yaml.template

```diff
@@ -20,16 +20,7 @@ build:
 
 test:
     requires:
-        - pytest
-        - pytest-flake8
-        - pytest-cov
-        - asynctest
-        - numpy
-        - astropy
-        - jsonschema
-        - pyyaml
-        - boto3
-        - moto
+        - ts-build
         - ts-dds
         - ts-idl
         - ts-salobj
```

Comment:
These last two lines should be

    - ts-idl {{ idl_version }}
    - ts-salobj {{ salobj_version }}